main/templates/main/base_template.html

```diff
@@ -27,7 +27,7 @@
                     <li class="nav-item"><a class="nav-link text-white" href="/profile">Profile</a></li>
                     <li class="nav-item"><a class="nav-link text-white" href="">Orders</a></li>
                     <li class="nav-item"><a class="nav-link text-white" href="">Search</a></li>
-                    <li class="nav-item"><a class="nav-link text-white" href="">Chats</a></li>
+                    <li class="nav-item"><a class="nav-link text-white" href="/messages">Chats</a></li>
```

Comment:
nit: I would call it chat

Reply:
name chat is occupied for chat between 2 users.
From '/messages' you can be redirected to '/chat' with a specific chat partner